pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -197,6 +198,20 @@ func (cm *containerManagerImpl) getNodeAllocatableAbsoluteImpl(capacity v1.Resou
 		}
 		result[k] = value
 	}
+
+	for k, v := range result {
```

Comment:
nodestatus in its setters.go is doing the same job. Since nodestatus uses cm package, maybe somehow make a common code in cm for it, because now this code duplication looks not so good.